src/javascript/static/pages/affiliates_faq.js

```diff
@@ -0,0 +1,6 @@
+const MenuSelector = require('../../_common/menu_selector');
+
+module.exports = {
+    onLoad  : () => { MenuSelector.init(['general', 'account-management-and-tracking', 'marketing-and-promotions', 'support']); },
+    onUnload: () => { MenuSelector.clean(); },
```

Comment:
seems a good candidate for https://github.com/binary-com/binary-static/blob/master/src/javascript/static/pages/static_pages.js instead of having its own file